### PR TITLE
Fix linter error and remove unused variables

### PR DESCRIPTION
**Pull request by Ingramz**

### CODE REVIEW

**Comment by winstliu** [comment]
Can you verify that fetch-property-docs still works properly after this change?  You could even submit a PR with those updates :wink:.

File: update.coffee [hunk 1]
```diff
@@ -31,7 +31,7 @@ Promise.settle([propertiesPromise, propertyDescriptionsPromise]).then (results)
     properties[propertyName] = metadata
     console.warn "No description for property #{propertyName}" unless propertyDescriptions[propertyName]?
 
-  for propertyName, d of propertiesRaw
+  for propertyName of propertiesRaw
```
Comment (winstliu) [inline update.coffee:34]: Should this be `in`?  Always forget the difference between the two.

**Comment by Ingramz** [comment]
Verified :smile:

**Comment by Ingramz** [comment]
@50Wliu ping.

**Comment by winstliu** [comment]
> @50Wliu ping.

Hey, give me some time!  7 hours isn't that long 😛 .

File: update.coffee (hunk 1 above)
Comment (Ingramz) [reply update.coffee:34]: In this case, I think not. propertyName is a key of object, d possibly meant description as value. (updated, was contradicting myself already)

Here's a handy trick to distinguish between the two. http://stackoverflow.com/questions/17886135/mnemonic-for-remembering-coffeescripts-in-vs-of

**Pull request merged**